testing/consul/APKBUILD

```diff
@@ -1,53 +1,29 @@
 # Contributor: Olivier Mauras <olivier@mauras.ch>
 # Maintainer:
 pkgname=consul
-pkgver=0.5.2
-pkgrel=1
+pkgver=0.6.4
+pkgrel=0
 pkgdesc="A tool for service discovery, monitoring and configuration"
 url="https://www.consul.io/"
 arch="all !aarch64"
 license="MPL 2.0"
 depends=""
 depends_dev=""
-makedepends="go godep perl bash mercurial $depends_dev"
+makedepends="go $depends_dev"
 install="$pkgname.pre-install $pkgname.pre-deinstall $pkgname.post-deinstall"
 pkgusers="consul"
 pkggroups="consul"
 subpackages=""
 options="!strip"
-source="http://dev.alpinelinux.org/archive/$pkgname/$pkgname-$pkgver.tar.gz
+source="$pkgname-$pkgver.tar.gz::https://github.com/hashicorp/$pkgname/archive/v$pkgver.tar.gz
 	consul.initd
 	consul.confd
 	acl.json.sample
 	encrypt.json.sample
 	server.json
 	tls.json.sample"
 
-_disturl="dev.alpinelinux.org:/archive/$pkgname/"
-_gourl="github.com/hashicorp/consul"
-_builddir="$srcdir"/src/github.com/hashicorp
-
-snapshot() {
-	abuild clean
-	abuild deps
-	abuild fetch
-	export GOPATH="$srcdir"
-	mkdir -p $_builddir
-	cd $_builddir
-	msg "Checking out v${pkgver} tag"
-	git clone -q --branch v${pkgver} https://$_gourl || return 1
-	cd $pkgname
-	# use custom godeps file
-	install -D "${_builddir}"/$pkgname/deps/v${pkgver//./-}.json \
-		$_builddir/$pkgname/Godeps/Godeps.json || return 1
-	go get -v -d || return 1
-	godep restore
-	cd "$srcdir"
-	tar zcf $pkgname-$pkgver.tar.gz src || return 1
-	rsync --progress -La $pkgname-$pkgver.tar.gz \
-		$_disturl || return 1
-	cd $startdir && abuild undeps
-}
+_builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
Could you please use the `builddir` name instead, and modify all the references to the `$_builddir` variable into `$builddir`?